### PR TITLE
chore: add basic type hints

### PR DESCRIPTION
**Pull request by scop**
This is correct after #11 has been applied, should be rebased on top of that re CHANGELOG.md anyway.

### CODE REVIEW

File: src/venvrun.py [hunk 1]
```diff
@@ -6,9 +6,10 @@
 import re
 import subprocess
 import sys
+from typing import List
```
Comment (guludo) [inline src/venvrun.py:9]: Could you please move this to the "from" import group?

**Review by guludo — CHANGES_REQUESTED** [review]
Thanks for this. The only thing I think should be changed is the location of the import for `List`.

File: src/venvrun.py (hunk 1 above)
Comment (guludo) [reply src/venvrun.py]: Nice. Github allowed me to do it 🙂 .

**Review by guludo — APPROVED** [review]

**Comment by guludo** [comment]
Hi, @scop. Thanks for this. I'll apply this now.

**Comment by scop** [comment]
Thanks. An isort config could help with communicating the preferred import order/style -- in this particular case there were two possibilities and I guessed wrong :)

**Comment by guludo** [comment]
Nice tip! Thanks. I had never heard of isort before.

**Pull request merged**